src/components/filter.js

```diff
@@ -78,21 +79,24 @@ const Filter = ({
             )
         }
       })}
-      <ListItem
-        innerDivStyle={style.listItemInner}
-        leftIcon={<SettingsIcon style={style.listIcon} />}
-        primaryText={<FormattedMessage {...messages.changeFilters} />}
+      <FlatButton
```

Comment:
Let's match the style of the list item and icon that was there before. I'm pedantic about layout/spacing.